File: JavaScript/7 kyu/Make a function that does arithmetic-.js
function arithmetic(a, b, operator){
    //your code here!
    switch(operator) {
      case "add":
        return a + b;
        break;
      case "subtract":
        return a - b;
        break;
      case "multiply":
        return a * b;
        break;
      case "divide":
        return a / b;
        break;
      default:
        return "Not a valid operator."
    }
  }

File: Python/7 kyu/Make a function that does arithmetic-.py
def arithmetic(a, b, operator):
    #your code here
    key = {
        \add\": \"+\"
        \"subtract\": \"-\"
        \"divide\": \"/\" 
        \"multiply\": \"*\"
    }
    
    return eval(f'{a} {key[operator]} b')"